SCSS/style.scss

```diff
@@ -0,0 +1,992 @@
+            &:hover {
+                cursor: pointer;
+            }
+            &:focus {
+                width: 150px;
+                opacity: 1;
+                cursor: text;
+            }
+            &:focus ~ .search {
+                right: -250px;
+                background: #151515;
+                z-index: 6;
+                &::before {
+                    top: 0;
+                    left: 0;
+                    width: 25px;
+                }
+                &::after {
+                    top: 0;
+                    left: 0;
+                    width: 25px;
+                    height: 2px;
+                    border: none;
+                    background: white;
+                    border-radius: 0%;
+                    transform: rotate(-45deg);
+                }
+            }
+            &::placeholder {
+                color: white;
+                opacity: 0.5;
+                font-weight: bolder;
+            }
+        }
+    }
+    // navbar ends 
+
+    // banner slider starts 
+
+ 
+
+    .banner-div{
+       .slick-slider{
+            .slick-dots{
+                bottom: -100px;
+                button::before{
+                    color: white;
+                }
+            }
+            .slick-next{
+                right:0px; 
+            }
+       }
+       .slick-arrow{
+        margin-top: 0vh;
+            &.slick-prev{
+                &::before {
+                    font-family: "Font Awesome 6 Free" !important ;
+                    content: '\f053' ;
+                    color: white ;
+                    font-weight: 900;
+                    font-size: 50px;
+                  }
+                @media only screen and (max-width:600px) {
+                    &::before {
+                    font-family: "Font Awesome 6 Free" !important ;
+                    content: '' ;
+                  
+                  }
+                }
+            }
+            
+            
+        &.slick-next::before {
+           
+            content: '\f054';
+            color: white;
+            font-weight: 900;
+            font-size: 50px;
+            font-family: "Font Awesome 6 Free";
+        
+     
+        }
+        @media only screen and (max-width:600px) {
+            &.slick-next::before  {
+            font-family: "Font Awesome 6 Free" !important ;
+            content: '' ;
+          
+          }
+    }
+        }
+        .inner-div{
+            width: 60%;
+            margin: 20vh auto;            
+            padding-top: 50px;
+            text-align: center;
+            color: white;
+            align-content: center;
+            h3{
+                font-size: 2rem;
+                font-weight: 300;
+            }
+            h1{
+                font-size: 3rem;
+            }
+            p{
+                width: 60%;
+                margin: 10px auto 50px auto;
+
+            }
+            @media only screen and (max-width:576px) {
```

Comment:
this is not the perfect way to use media query please fix this

Reply:
done